### PR TITLE
PLT-8198: System Console updates for message/compliance export

### PR DESCRIPTION
**Pull request by MusikPolice**
#### Summary
Updated the text on the System Console settings page to approved copy

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-8198

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Has UI changes
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates

### CODE REVIEW

File: components/admin_console/message_export_settings.jsx [hunk 1]
```diff
@@ -37,40 +37,40 @@ export default class MessageExportSettings extends AdminSettings {
...
+                            defaultMessage='Enable Compliance Export:'
                         />
                     }
                     helpText={
-                        <FormattedMessage
-                            id='admin.service.messageExportDesc'
-                            defaultMessage='When true, the system will export all messages that are sent once per day.'
+                        <FormattedHTMLMessage
+                            id='admin.service.complianceExportDesc'
+                            defaultMessage='When true, Mattermost will generate a compliance export file that contains all messages that were posted in the past 24h. The export task is scheduled to run once per day. See <a href=\"https://about.mattermost.com/default-compliance-export-documentation\" target=\"_blank\">the documentation</a> to learn more.'
```
Comment (jasonblais) [inline components/admin_console/message_export_settings.jsx]: `in the past 24h` --> `in the last 24 hours`.

File: components/admin_console/message_export_settings.jsx [hunk 2]
```diff
@@ -81,15 +81,15 @@ export default class MessageExportSettings extends AdminSettings {
...
                     }
-                    placeholder={Utils.localizeMessage('admin.messageExport.exportJobStartTime.example', 'E.g.: "02:00"')}
+                    placeholder={Utils.localizeMessage('admin.complianceExport.exportJobStartTime.example', 'E.g.: "02:00"')}
                     helpText={
-                        <FormattedMessage
-                            id='admin.messageExport.exportJobStartTime.description'
-                            defaultMessage='Set the start time of the daily scheduled message export job. Choose a time when fewer people are using your system. Must be a 24-hour time stamp in the form HH:MM.'
+                        <FormattedHTMLMessage
+                            id='admin.complianceExport.exportJobStartTime.description'
+                            defaultMessage='Set the start time of the daily scheduled compliance export job. Choose a time when fewer people are using your system.<br />Must be a 24-hour time stamp in the form HH:MM. For example, 1:15am would be expressed as 01:15, while 1:15pm would be expressed as 13:15.'
```
Comment (jasonblais) [inline components/admin_console/message_export_settings.jsx]: `<br />Must be a 24-hour time stamp in the form HH:MM. For example, 1:15am would be expressed as 01:15, while 1:15pm would be expressed as 13:15.` --> `Must be a 24-hour time stamp in the form HH:MM.`

I.e. no need for an example (the placeholder has an example) nor a need for a new line

File: components/admin_console/message_export_settings.jsx [hunk 3]
```diff
@@ -103,13 +103,13 @@ export default class MessageExportSettings extends AdminSettings {
...
-                            id='admin.messageExport.exportFormat.title'
+                            id='admin.complianceExport.exportFormat.title'
                             defaultMessage='Export File Format:'
                         />
                     }
                     helpText={
                         <FormattedMessage
-                            id='admin.messageExport.exportFormat.description'
+                            id='admin.complianceExport.exportFormat.description'
                             defaultMessage='The file format to write exported data in. Corresponds to the system that you want to import the data into.'
```
Comment (jasonblais) [inline components/admin_console/message_export_settings.jsx]: Propose `File format of the compliance export.`

File: components/admin_console/message_export_settings.jsx [hunk 4]
```diff
@@ -143,14 +143,14 @@ export default class MessageExportSettings extends AdminSettings {
                     disabled={!this.state.enableMessageExport}
                     createJobButtonText={
                         <FormattedMessage
-                            id='admin.messageExport.createJob.title'
-                            defaultMessage='Run Message Export job now'
+                            id='admin.complianceExport.createJob.title'
+                            defaultMessage='Run Compliance Export job now'
```
Comment (jasonblais) [inline components/admin_console/message_export_settings.jsx]: Write in camel case (`Run Compliance Export Job Now`)

File: components/admin_console/message_export_settings.jsx [hunk 5]
```diff
@@ -37,40 +37,40 @@ export default class MessageExportSettings extends AdminSettings {
...
+                            defaultMessage='Enable Compliance Export:'
                         />
                     }
                     helpText={
-                        <FormattedMessage
-                            id='admin.service.messageExportDesc'
-                            defaultMessage='When true, the system will export all messages that are sent once per day.'
+                        <FormattedHTMLMessage
+                            id='admin.service.complianceExportDesc'
+                            defaultMessage='When true, Mattermost will generate a compliance export file that contains all messages that were posted in the last 24h. The export task is scheduled to run once per day. See <a href=\"https://about.mattermost.com/default-compliance-export-documentation\" target=\"_blank\">the documentation</a> to learn more.'
```
Comment (jasonblais) [inline components/admin_console/message_export_settings.jsx]: `24h` --> `24 hours`

File: components/admin_console/message_export_settings.jsx [hunk 6]
```diff
@@ -103,14 +103,14 @@ export default class MessageExportSettings extends AdminSettings {
...
+                            id='admin.complianceExport.exportFormat.title'
                             defaultMessage='Export File Format:'
                         />
                     }
                     helpText={
                         <FormattedMessage
-                            id='admin.messageExport.exportFormat.description'
-                            defaultMessage='The file format to write exported data in. Corresponds to the system that you want to import the data into.'
+                            id='admin.complianceExport.exportFormat.description'
+                            defaultMessage='The file format of the compliance export. Corresponds to the system that you want to import the data into.'
```
Comment (jasonblais) [inline components/admin_console/message_export_settings.jsx]: `The file format` --> `File format`

**Comment by MusikPolice** [comment]
Huh, [this build passed](https://newbuild.mattermost.com/job/mattermost-webapp/view/change-requests/job/PR-390/4/console), but the GitHub status isn't updating

**Comment by ccbrown** [comment]
@MusikPolice eslint error:

```
$ eslint --ext ".jsx" --ignore-pattern node_modules --quiet .

/home/jenkins/workspace/mattermost-webapp_PR-390-36NXF5CNWKI3E2N5AKJKUN5KI264AOHRKIOLEBWY2PCC4EY3XHQA/mattermost-webapp/components/admin_console/message_export_settings.jsx
  31:74  error  Unexpected trailing comma  comma-dangle

✖ 1 problem (1 error, 0 warnings)
```

**Comment by asaadmahmood** [comment]
@MusikPolice Can we get a test server for this?

**Comment by ccbrown** [comment]
@MusikPolice Needs a rebase.

**Pull request merged**